Data/ByteString/Short/Internal.hs

```diff
@@ -160,6 +160,9 @@ import Data.ByteString.Internal.Type
   , unsafeDupablePerformIO
   , accursedUnutterablePerformIO
   , checkedAdd
+  , c_elem_index
+  , cIsValidUtf8BASafe
+  , cIsValidUtf8BA
```

Comment:
Let's not add these exports.

Reply:
I don't understand. These are imports. Or do you mean moving the definitions in another module?

Reply:
Whoops, sorry for the noise. Reading is hard.